api_test.go

```diff
@@ -175,13 +175,6 @@ func (NodeProvider) makeAPISwarm(ctx context.Context, fullIdentity bool, n int)
 				errs <- err
 				return
 			}
-
-			// empty node is pinned even with --empty-repo, we don't want that
```

Comment:
this is just in test code?

Reply:
Yes. But I was wrong, this isn't the issue. I'm trying to figure out why these tests are failing when running against go-ipfs master.